Examples/create-app.js

```diff
@@ -61,6 +61,17 @@ let iosStagingDeploymentKey = null;
 createCodePushApp(appNameAndroid, 'Android');
 createCodePushApp(appNameIOS, 'iOS');
 
+if (!isReactNativeVersionLowerThan(60) && process.platform === "darwin") {
+  try {
+    console.log("Verify that CocoaPods installed");
+    execCommand("pod --version");
+  } catch {
+    console.log("Need to install CocoaPods");
+    execCommand("sudo gem install cocoapods");
```

Comment:
Will it work if node process started from non root?

If not probably we can just ask the user to install and stop further execution for script.

Reply:
It works if node process started from non root but process will ask enter password.

Reply:
I'm not sure that we should call installing command. In my opinion, @snowpardx's approach is more correct...

Reply:
Ok. Done